packages/lesswrong/components/messaging/FriendlyConversationItem.tsx

```diff
@@ -57,7 +64,13 @@ const styles = (theme: ThemeType): JssStyles => ({
     fontSize: "13px",
     fontWeight: 500,
     marginTop: 2,
-  }
+  },
+  unread: {
```

Comment:
All of these styling changes are probably provisional until @agnesstenlund can take a look.